charts/horizon/templates/horizon-ingest-cm.yaml

```diff
@@ -20,7 +20,10 @@ data:
   TMPDIR: "/var/lib/stellar"
   CAPTIVE_CORE_STORAGE_PATH: "/var/lib/stellar"
   CAPTIVE_CORE_REUSE_STORAGE_PATH: "True"
+  CAPTIVE_CORE_REUSE_STORAGE_DIR: "True"
   CAPTIVE_CORE_USE_DB: {{ .Values.ingest.horizonConfig.captiveCoreUseDb | default "false" | quote}}
+  ENABLE_EXPERIMENTAL_INGESTION: "false"
+  FRIENDBOT_URL: "https://friendbot.stellar.org"
```

Comment:
It would be good to make this a parameter. We could default to friendbot.stellar.org

Reply:
Done!